src/TableView.elm

```diff
@@ -314,9 +318,13 @@ viewCell dateWithSessions model timeDelimiters numColumns timeDelimiter index co
                 |> Maybe.withDefault -1
 
         trackId =
-            sessionStarting
-                |> Maybe.map .trackId
-                |> Maybe.withDefault 0
+            case (Maybe.map .trackId sessionStarting) of
```

Comment:
duplication here. Also we can use [andThen](http://package.elm-lang.org/packages/elm-lang/core/latest/Maybe#andThen) to do this more succinctly